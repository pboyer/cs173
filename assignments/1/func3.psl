defvar x = "" in
defvar f = lambda(y) { x += +("f says: ", y, "\n"); } in
defvar g = lambda(y) { x += +("g says: ", y, "\n"); } in {
  f("hello!");
  g("did you add that?");
  f("who let you add to this string?");
  x;
}