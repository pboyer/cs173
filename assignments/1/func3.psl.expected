f says: hello!
g says: did you add that?
f says: who let you add to this string?

